config/config.json.dist

```diff
@@ -11,7 +11,8 @@
     "files",
     "search",
     "media-viewer",
-    "draw-io"
+    "draw-io",
+    "user-management"
```

Comment:
Should we enable the user-management app per default? Then it also should go into the config.json.dist.ocis (or sth, next to this file) and to the oCIS configs in `dev/docker/` and `tests/drone/` :)

Reply:
> Should we enable the user-management app per default? 
yes